components/search/MoreFacetsDropdownOptions.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <b-dropdown-form>
+    <strong
+      class="mb-4 d-inline-block"
+    >
+      {{ $t(`facets.${facet.name}`).name }}
+    </strong>
+    <div class="option-group">
+      <MoreFacetsDropdownOption
+        v-for="(filter, indx) in [].concat(facet.fields).splice(0, limitTo)"
+        :key="indx"
+        :filter="filter"
+        :facet-name="$t(`facets.${facet.name}`).name"
+      />
+    </div>
+    <div
+      v-if="facet.fields.length > limitTo"
+      v-show="isActive"
+      class="option-group"
+    >
+      <MoreFacetsDropdownOption
+        v-for="(filter, indx) in [].concat(facet.fields).splice(limitTo)"
+        :key="indx"
+        :filter="filter"
+        :facet-name="$t(`facets.${facet.name}`).name"
+      />
+    </div>
+    <button
+      v-if="facet.fields.length > limitTo"
+      type="button"
+      class="btn btn-link btn-toggle"
+      :class="{ 'is-active': isActive }"
+      :data-qa="(isActive ? $t(`facets.button.hideAll`) : $t(`facets.button.showAll`)) + ' ' + $t(`facets.${facet.name}`).plural + ' button'"
```

Comment:
We can not assume that the facet label will come at the end of the localised string. as in different languages it may not. The localised string should have a placeholder for the facet label.

Reply:
from what I can see, Nuxt i18n doesn't support Pluralization :(

Reply:
That was presumably in response to https://github.com/europeana/portal.js/pull/331#discussion_r344130012

Reply:
Yes sorry.